src/main/resources/assets/not_enough_gamerules/lang/en_us.json

```diff
@@ -6,6 +6,9 @@
   "gamerule.anvilDamage": "Deal anvil damage",
   "gamerule.canCopperOxide": "Can copper oxide",
   "gamerule.canCopperOxide.description": "Defines if normal copper can oxidize",
+  "gamerule.canEntityInteractWithBlock": "Can entity interact with blocks",
+  "gamerule.canEntityInteractWithEntity": "Can entity interact with another entities",
```

Comment:
Not very English, either "interact with other entities", or "interact with another entity", but can't be both. You need to choose one.

Reply:
I realized that and was going to make a commit to change that